### PR TITLE
[Chat Sdk for Android]: Update README.md and CHANGELOG.md for publishing

### PR DESCRIPTION
**Pull request by JianpingChen**

### CODE REVIEW

File: sdk/communication/azure-communication-chat/README.md [hunk 1]
```diff
@@ -80,7 +80,9 @@ The id for the user created above will be necessary later to add said user as a
 
 ## Chat Thread
 
-A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat.
+
+Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
 
 # Chat operations
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:87]: I think you also need to update the version to 1.0.0-beta.2

**Review by ashwinder — APPROVED** [review]
readme changes approved

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.3 (Unreleased)
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:3]: I think beta 3 should be added after the release.

File: sdk/communication/azure-communication-chat/README.md [hunk 3]
```diff
@@ -80,7 +80,9 @@ The id for the user created above will be necessary later to add said user as a
 
 ## Chat Thread
 
-A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat.
+
+Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-chat/README.md:85]: We do not have real-time update yet. Instead let's say "Using the APIs, users can also send typing indicators when typing a message and read receipts for the messages they have read in a chat thread. To learn more, read about chat service here." and then link here to the chat concept doc

Comment (JianpingChen) [reply sdk/communication/azure-communication-chat/README.md:85]: "and read receipts for the messages they have read". Do you mean "and **send** receipts for the messages they have read"?

**Review by RezaJooyandeh — APPROVED** [review]

**Review by anuchandy — APPROVED** [review]

**Pull request merged**